@types/jsb.d.ts

```diff
@@ -363,3 +363,8 @@ declare namespace ns {
     export class Frustum extends jsb.NativePOD {
     }
 }
+
+/**
+ * Only defined on native platforms.
+ */
+declare const scriptEngineType: 'JavaScriptCore' | 'SpiderMonkey' | 'V8';
```

Comment:
Only support v8 now.

Reply:
we still use 'JavaScriptCore' in debug.ts

Reply:
here I new an issue https://github.com/cocos/3d-tasks/issues/15940